server/src/main/java/org/loomdev/loom/command/CommandSourceImpl.java

```diff
@@ -22,6 +25,11 @@ public CommandSourceImpl(net.minecraft.commands.CommandSource source) {
         this.source = source;
     }
 
+    @NotNull
+    public net.minecraft.commands.CommandSource getMinecraftSource() {
+        return source;
+    }
+
```

Comment:
I don't know why this was added, since nothing uses it?

Reply:
I needed it at one point and forgot to remove it.